app/views/layouts/devise.html.erb

```diff
@@ -2,7 +2,7 @@
   <%= content_for(:header) %>
 
   <div class="container">
-    <div class="mr-auto ml-auto w-25 mt-5 mb-5">
+    <div class="col-md-8 offset-md-2 devise-wrap">
```

Comment:
the fields look super long to me at this size. what if we did `col-md-6 offset-md-3`?

Reply:
The original design included links both left- and right-aligned below the horizontal rule, and if I remember correct in some cases (e.g., the Forgot your password? page) going smaller than `col-md-8` put the adjacent links too close together to look good. And there's no harm in an overly wide input field on this page so that seemed the best compromise.

I guess if we're going to stick to left-aligned only links, the above situation is no longer in play, so sure, I guess we could go with `col-md-6 offset-md-3` as long as that still works at all breakpoints.

Reply:
Oops, when I commented my view hadn't updated since Gary's comment.

Reply:
@ggeisler thx for context. i think let's just commit to keeping everything left-aligned so we can shrink the overall container a bit on large screens.